docs/src/parameter_distributions.md

```diff
@@ -16,9 +16,14 @@ Solution: We should use a Normal distribution with the predefined "bounded" cons
 
 Let's initialize the constraint first,
 ```julia
-constraint = [bounded(0,1)] # Sets up a logit-transformation into [0,1].
+constraint = [bounded(0,1)] 
 ```
-The prior is around 0.7, and the push forward of a normal distribution N(mean=1,sd=0.5) gives a prior with 95% of it's mass between [0.5,0.88].
+This sets up a logit transformation to and from the constrained space
+```julia
+unconstrained_to_constrained(x) = exp(x) / (exp(x) + 1)
+constrained_to_unconstrained(x) = log( x / (1 - x))
+```
+The prior is around 0.7 (constrained space), and the push-forward of an unconstrained normal distribution N(mean=1,sd=0.5) through the logit transformation gives a prior with 95% of it's mass between [0.5,0.88].
```

Comment:
Was the push-forward defined somewhere else in the docs?

Reply:
typo: "it's" should be "its"
And isn't the normal distribution sent through a sigmoid transformation (rather than its inverse, the logit transformation)?

Reply:
@trontrytel By pushforward I mean you apply the function `unconstrained_to_constrained(Normal(1,0.5))` to it

@bielim thanks - I'm going to just not mention it,